arcgis-ios-sdk-samples/Content Display Logic/Storyboard/Main.storyboard

```diff
@@ -348,41 +344,69 @@
                         <subviews>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="rA3-JQ-Mdx" userLabel="Content View">
-                                <rect key="frame" x="0.0" y="0.0" width="375" height="680"/>
-                                <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                            </view>
-                            <toolbar opaque="NO" clipsSubviews="YES" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="cZS-cc-Di0">
-                                <rect key="frame" x="0.0" y="680" width="375" height="44"/>
-                                <color key="backgroundColor" red="0.97647720575332642" green="0.9764588475227356" blue="0.97647124528884888" alpha="1" colorSpace="custom" customColorSpace="displayP3"/>
+                                <rect key="frame" x="0.0" y="0.0" width="414" height="764"/>
+                                <subviews>
+                                    <wkWebView contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="KE4-Ic-Gt2">
+                                        <rect key="frame" x="0.0" y="0.0" width="414" height="764"/>
+                                        <wkWebViewConfiguration key="configuration">
+                                            <audiovisualMediaTypes key="mediaTypesRequiringUserActionForPlayback" none="YES"/>
+                                            <wkPreferences key="preferences"/>
+                                        </wkWebViewConfiguration>
+                                    </wkWebView>
+                                </subviews>
                                 <constraints>
-                                    <constraint firstAttribute="height" constant="44" id="Kff-Ig-9pK"/>
+                                    <constraint firstAttribute="bottom" secondItem="KE4-Ic-Gt2" secondAttribute="bottom" id="ApL-oy-iHl"/>
+                                    <constraint firstItem="KE4-Ic-Gt2" firstAttribute="leading" secondItem="rA3-JQ-Mdx" secondAttribute="leading" id="a3B-K1-Iw3"/>
+                                    <constraint firstAttribute="trailing" secondItem="KE4-Ic-Gt2" secondAttribute="trailing" id="nxz-eX-LGE"/>
+                                    <constraint firstItem="KE4-Ic-Gt2" firstAttribute="top" secondItem="rA3-JQ-Mdx" secondAttribute="top" id="sqb-Ou-quV"/>
                                 </constraints>
+                            </view>
+                            <toolbar opaque="NO" clipsSubviews="YES" clearsContextBeforeDrawing="NO" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="cZS-cc-Di0">
+                                <rect key="frame" x="0.0" y="764" width="414" height="44"/>
```

Comment:
There is an issue with the toolbar bottom constraints right now. Since the Source Code View Controller is embedded in a container view, storyboard fails to snap the safe area with its superview.

![image](https://user-images.githubusercontent.com/9660181/95800689-da38dd00-0cac-11eb-9f02-3d6c7870b45e.png)

I've asked for help from our instructor in last training, but feel free to give me some help/thoughts!

Reply:
I'm not seeing that with an iOS 14 simulator. Which simulator are you using?

![image](https://user-images.githubusercontent.com/2257493/96197231-47937a80-0f06-11eb-9aac-6eb3be96a802.png)

Reply:
Never mind. I didn't see that some changes had been force pushed. After resetting my local branch, I now see the issue. I'll take a look.

Reply:
Sorry for the force push. There were quite a few conflicts and I decided to ditch the outdated changes and redo from scratch. 

The previous version set the container view's background to the same as the toolbar's bar tint to hide this problem, but that won't work under Dark Mode.

The workaround that I came up with is to 

1. Uncheck the translucent checkbox of toolbar, so that it doesn't have the UIVisualEffect view for translucency
2. Set its bar tint to the same as the base/root view's background color, such as `systemBackgroundColor`. 

However, toolbars in samples or in source code pane shouldn't be different in colors, so I didn't make this change.

Reply:
Solved by making a small tweak in c27369e 😏

Reply:
|Translucent|Opaque|
|-|-|
|![image](https://user-images.githubusercontent.com/9660181/96792919-363be980-13b0-11eb-80fc-ea0593f737c8.png)|![image](https://user-images.githubusercontent.com/9660181/96793046-70a58680-13b0-11eb-8d02-7a7a32c6c343.png)|

One more thing to mention: while the purple tint looks beautiful on an opaque bar, it might be a bit bright on a translucent bar. Should I tweak the accentColor for Dark Appearance?

Reply:
Yeah, looks like that might be a good idea.

Reply:
Slightly adjusted the purple color in Dark Mode
|Before|After|
|-|-|
|![](https://user-images.githubusercontent.com/9660181/96792919-363be980-13b0-11eb-80fc-ea0593f737c8.png)|![image](https://user-images.githubusercontent.com/9660181/96924688-dd2f8c80-1467-11eb-864b-8767892e2525.png)|